h2o-core/src/main/java/water/Scope.java

```diff
@@ -85,12 +85,11 @@ static private void track_impl(Scope scope, Key key) {
       scope._keys.peek().add(key);            // Track key
   }
 
-  static public void untrack( Key<Vec>[] keys ) {
+  static public void untrack(Iterable<Key<Vec>> keys) {
```

Comment:
I do not understand the change of API - if you need a new method, you can always provide overloaded alternative, but removing the method totally is asking for problems. Especially, when external developers depend on it...